.commitlintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+	"extends": [
+		"@commitlint/config-conventional"
+	],
+	"helpUrl": "https://perunaai.atlassian.net/wiki/spaces/STRIBOG/pages/13467868/Semantic+versioning",
+	"rules": {
+		"scope-enum": [
+			2,
+			"always",
+			[
+				"core",
+				"openapi",
+				"cli",
+				"gui",
+				"engine",
+				"registrar",
+				"dispatcher",
+				"mvn",
+				"update"
```

Comment:
This scope was added, so this PR wouldn't fail. It should be removed before merge.